lib/course_planner_web/templates/task/show.html.eex

```diff
@@ -33,15 +33,15 @@
                 <div class="detail">
                   <div class="detail__name">Start time</div>
                   <div class="detail__value">
-                    <%= String.slice NaiveDateTime.to_iso8601(@task.start_time), 0, 10 %>
-                    <%= String.slice NaiveDateTime.to_iso8601(@task.start_time), 11, 5 %>
+                    <%= String.slice NaiveDateTime.to_iso8601(@task.start_time |> Settings.utc_to_system_timezone()), 0, 10 %>
```

Comment:
as mentioned above, time set by user is always set according to her own timezone. we don't need conversion on it. or unless it will be double conversion

Reply:
It should be converted to UTC to store and converted back to the configured timezone when displaying